R/rdeephaven/R/helper_functions.R

```diff
@@ -0,0 +1,121 @@
+################ FUNCTIONS TO GET FROM DH TO R
```

Comment:
I agree with you. This is where being unable to access the S4 internals in the source code is a pain. Since these require R functionality, they can't be walled up in the C++, and the proper methods cannot get added to the S4 without access.